### ISSUE TITLE
Implement Learning rate schedulers

### ISSUE THREAD

**Issue by mikkelfo** [FGA-DIKU/EHR#339]
Ideas might be: Linear warmup, cosine annealing, warmup-stable-decay (for pretraining!)

**Comment by mikkelfo** [comment]
Linear warmup (`LinearLR`) is implemented in #345
